### DISCUSSION TITLE
Social links are blocking the docs links

### DISCUSSION THREAD

**Discussion by wdfinch** [metalbear-co/mirrord#2075] [General]
Social links are blocking the official website sidebar links. This makes it really hard to use the docs. Can this please be fixed? 

<img width="1298" alt="Screenshot 2023-11-15 at 9 02 56 AM" src="https://github.com/metalbear-co/mirrord/assets/36898266/c12803ec-ef33-49e2-a76f-d873856c31b1">

**Comment by eyalb181** [comment]
Whoops! Thanks for reporting this. Should be fixed now.
